.github/workflows/docker.yml

```diff
@@ -0,0 +1,88 @@
+name: Create and publish a Docker image
```

Comment:
Perhaps more descriptive might be, "CI/CD Docker Image Build and Publish"

Reply:
This is technically just the CI part. I don't think it's too necessary either way.